plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/font-pairing-variations/preview.tsx

```diff
@@ -159,6 +166,10 @@ export const FontPairingVariationPreview = () => {
 						</HStack>
 					</div>
 				</div>
+				<FontFamiliesLoader
```

Comment:
Adds the font loading to the iframes, similar to [the original](https://github.com/Automattic/wp-calypso/blob/d3c9b16fb99ce242f61baa21119b7c20f8823be6/packages/global-styles/src/components/font-pairing-variations/preview.tsx#L139).